### ISSUE TITLE
Plug-in Installed, now blank screens

### ISSUE THREAD

**Issue by jhough-zz** [vincentsels/ProjectManagement#15]
Hello,

I installed the plug-in on to our server this afternoon. Clicked the install link in the management section. Now all pages are white. I have renamed the plug-in directory and Mantis is working again.

How can disable/uninstall the plug-in without the manage page?

More importantly, we'd really like to use the plug-in. What am I missing? We are using the most current mbt and the plugin posted here.

Thanks,
Jeff

**Comment by jhough-zz** [comment]
Any thoughts on this? We'd like to use the plug-in. I can't even uninstall...

Thanks,
Jeff

**Comment by vincentsels** [comment]
First of all, sorry for the late response, I don't recall having received an email notification for this issue - strange. I just pulled the latest Mantis version (1.2.12-dev), uninstalled the plugin, removed the plugin folder entirely and reinstalled the latest version. Can't reproduce it. Tried in Chrome, IE and Firefox (I'm certain Firefox didn't have anything cashed yet). Possibly has got something to do with Mantis or php configuration. 
- If you don't mind, perhaps you could mail me your config_inc.php file (you can leave out sensitive information of course), and I could try with that...
- I'm not a php expert either, so if that's causing the problem, it'll be difficult for me to find out :( Do you see anything that might be of use in your php/mysql logs ?

In order to fully uninstall the plugin, apart from removing the directory, you need to clear two entries from the database: 
delete from mantis_plugin_table where basename = 'ProjectManagement'
delete from mantis_config_table where config_id = 'plugin_ProjectManagement_schema'

**Comment by jhough-zz** [comment]
No problem. I have posted an archive of the complete file set here: http://www.sitecraft.com/otherstuff/clients/first_insight/mantis_test/mantis_test.zip

I have omitted the sensitive info.

I'll wait to hear from you before I uninstall. Maybe it is simply a tweak.

**Comment by vincentsels** [comment]
jhough, I tried using your complete folder, only changed username, password and database. Created new blank database, and installed the plug-in. I got an error saying there was a problem with the english translation file - did you get and fix that ? If not, I hope that's the problem. I already checked it in on the master branch. 

If that doesn't fix it, I can't reproduce it (yet)...

Have you looked at the html source of the blank page; does it contain anything ? Or did you find anything in any (php/mysql) log ?
